src/constants.ts

```diff
@@ -3,12 +3,15 @@ import { WalletStatEntry } from './inferno_types';
 export const OUTPUT_TEMPLATE_NAME = 'daily_output.base.json';
 export const OUTPUT_NAME = 'daily_output.json';
 export const NON_UNBUNDLED_BUNDLES_NAME = 'non_unbundled_bundles.json';
-export const ONE_THOUSAND_MB = 1000 * 1000 * 1000;
-export const GROUP_EFFORT_REWARDS = [
+export const ONE_HUNDRED_MB = 1000 * 1000 * 100;
+export const GROUP_EFFORT_REWARDS_FULL = [
 	75, 55, 50, 47, 45, 43, 40, 37, 35, 32, 31, 30, 29, 28, 27, 26, 25, 24, 23, 22, 21, 20, 19, 18, 17, 16, 15, 14, 13,
 	12, 11, 10, 9, 9, 8, 8, 7, 7, 6, 6, 5, 5, 4, 4, 3, 3, 2, 2, 1, 1
 ] as const;
-export const GQL_URL = 'https://arweave.net/graphql';
+export const GROUP_EFFORT_REWARDS_HALF = [
+	38, 28, 25, 24, 23, 22, 20, 19, 18, 16, 16, 15, 14, 14, 13, 13, 12, 12, 11, 11, 10, 10, 9, 9, 8, 8, 7, 7, 6, 6, 5,
+	5, 5, 4, 4, 4, 4, 3, 3, 3, 3, 2, 2, 2, 2, 1, 1, 1, 1, 1
+] as const;
```

Comment:
```ts
for (const rewardIndex in GROUP_EFFORT_REWARDS_FULL) {
    const reward = GROUP_EFFORT_REWARDS_FULL[rewardIndex];
    const expectedHalfReward = Math.ceil(reward / 2);
    const halfReward = GROUP_EFFORT_REWARDS_HALF[rewardIndex];
    if (expectedHalfReward !== halfReward) {
        console.log(`Missmatch at index ${rewardIndex} (${reward}): expected:${expectedHalfReward} => actual:${halfReward}`);
 }
}

/**
Missmatch at index 12 (29): expected:15 => actual:14
Missmatch at index 14 (27): expected:14 => actual:13
Missmatch at index 16 (25): expected:13 => actual:12
Missmatch at index 18 (23): expected:12 => actual:11
Missmatch at index 20 (21): expected:11 => actual:10
Missmatch at index 22 (19): expected:10 => actual:9
Missmatch at index 24 (17): expected:9 => actual:8
Missmatch at index 26 (15): expected:8 => actual:7
Missmatch at index 28 (13): expected:7 => actual:6
Missmatch at index 30 (11): expected:6 => actual:5
Missmatch at index 33 (9): expected:5 => actual:4
Missmatch at index 37 (7): expected:4 => actual:3
Missmatch at index 41 (5): expected:3 => actual:2
Missmatch at index 45 (3): expected:2 => actual:1
**/
```
There are some inconsistencies here, we actually want the output of this:
```ts
GROUP_EFFORT_REWARDS_FULL.map(reward => Math.ceil(reward / 2));
```

```suggestion
export const GROUP_EFFORT_REWARDS_HALF = [
	38, 28, 25, 24, 23, 22, 20, 19, 18, 16, 16, 15, 15, 14, 14, 13, 13, 12, 12, 11, 11, 10, 10, 9, 9, 8, 8, 7, 7, 6, 6,
	5, 5, 5, 4, 4, 4, 4, 3, 3, 3, 3, 2, 2, 2, 2, 1, 1, 1, 1
] as const;
```

Reply:
We can either have it hardcoded as it is, we can have the mapping, or/and we can have a test to test for it

Reply:
Any of both suggestions are good to me
```suggestion
export const GROUP_EFFORT_REWARDS_HALF = GROUP_EFFORT_REWARDS_FULL.map(reward => Math.ceil(reward / 2));
```

Reply:
I'll check the requirements again and make a decision.
Thank you!

Reply:
Nice